[venus-sector-manager] 没有加载v8 builtin actor引起'loading miner state: unknown actor'错误

### 模块 / Components

- [X] venus-sector-manager
- [ ] venus-worker
- [ ] 工具链 / toolchains
- [ ] 文档 / docs

### 版本 / Version

```text
venus-sector-manager -v
venus-sector-manager version v0.3.0-rc2-prod-git.85b3858
```


### 描述 / Describe the Bug

任务报错，无法恢复
sector_id=Some(s-t0xxxx-2354), paused=true, paused_elapsed=Some(1385s), state=C2Done, last_err=Some("permanent: rpc error: JsonRpcError(Error { code: ServerError(1), message: \"get sector precommit info failed\", data: None })")
<img width="1750" alt="image" src="https://user-images.githubusercontent.com/43364105/174255896-c676784e-a84b-4de1-ae5a-e2d8fca575df.png">


### 日志 / Logging Information

```text
2022-06-17T15:47:56.430+0800    WARN    rpc     go-jsonrpc@v0.1.4/handler.go:373        error in RPC call to 'Venus.SubmitPreCommit': getting precommit info: handleSealFailed(2355): temp error: loading miner state: unknown actor code bafk2bzacea6rabflc7kpwr6y4lzcqsnuahr4zblyq3rhzrrsfceeiw2lufrb4
```


### 重现步骤 / Repo Steps

_No response_

--- a/venus-sector-manager/modules/policy/const.go
+++ b/venus-sector-manager/modules/policy/const.go
@@ -6,6 +6,8 @@ import (
 	"github.com/filecoin-project/venus/fixtures/networks"
 	"github.com/filecoin-project/venus/venus-shared/actors/builtin"
 	"github.com/filecoin-project/venus/venus-shared/actors/policy"
+	builtinactors "github.com/filecoin-project/venus/venus-shared/builtin-actors"
+	"github.com/filecoin-project/venus/venus-shared/types"
 
 	"github.com/ipfs-force-community/venus-cluster/venus-sector-manager/pkg/logging"
 )
@@ -16,26 +18,34 @@ var NetParams *networks.NetworkConf
 
 func SetupNetwork(name string) error {
 	log.Debugw("NETWORK SETUP", "name", name)
+	var networkType types.NetworkType
 	switch name {
 	case "mainnet":
 		NetParams = networks.Mainnet()
+		networkType = types.NetworkMainnet
 	case "integrationnet":
 		NetParams = networks.IntegrationNet()
+		networkType = types.Integrationnet
 	case "2k":
 		NetParams = networks.Net2k()
+		networkType = types.Network2k
 	case "cali":
 		NetParams = networks.Calibration()
+		networkType = types.NetworkCalibnet
 	case "interop":
 		NetParams = networks.InteropNet()
+		networkType = types.NetworkInterop
 	case "force":
 		NetParams = networks.ForceNet()
+		networkType = types.NetworkForce
 	case "butterfly":
 		NetParams = networks.ButterflySnapNet()
+		networkType = types.NetworkButterfly
 	default:
 		return fmt.Errorf("invalid network name %s", name)
 	}
 
-	return nil
+	return builtinactors.SetNetworkBundle(networkType)
 }
 
 const (

